app/grandchallenge/challenges/emails.py

```diff
@@ -42,3 +44,133 @@ def send_external_challenge_created_email(challenge):
         subject=f"[{site.domain.lower()}] New External Challenge",
         message=message,
     )
+
+
+def send_challenge_requested_email_to_reviewers(challengerequest):
+    site = Site.objects.get_current()
+    update_url = reverse(
+        "challenges:request-list",
+    )
+    message = (
+        f"Dear reviewers,\n\n"
+        f"User {challengerequest.creator} has just requested the challenge "
+        f"{challengerequest.title}. To review the challenge, go here:"
+        f"{update_url}\n\n"
+        f"Regards,\n"
+        f"{site.name}\n\n"
+        f"This is an automated service email from {site.domain}."
+    )
+    reviewers = (
+        Group.objects.filter(name=settings.CHALLENGE_REVIEWERS_GROUP_NAME)
```

Comment:
Rather than using the group look for users who have the global permission (either directly or through a group), django guardian has methods to do that.